README.md

```diff
@@ -1,3 +1,3 @@
 # Optikon DNS
 
-Custom DNS for Optikon, Using Homemade CoreDNS Plugins
+Custom DNS for Optikon, Using Homemade CoreDNS Plugins.
```

Comment:
Lets buff this up a little, we can throw one of the diagrams we used on the presentation.

Reply:
I'll take a shot at it and submit a PR for this branch. Please correct whatever I submit and feel free to add more detail